transport/core/config.go

```diff
@@ -0,0 +1,42 @@
+package core
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+)
+
+// TLSClientAuthClientConfig returns a new client authentication TLS
+// configuration that can be used for a client using client auth
+// connecting to the named host.
+func TLSClientAuthClientConfig(cert tls.Certificate, host string) *tls.Config {
```

Comment:
I've since moved these over the the `transport` package and `git rm`'d this file (collecting comments before a commit and push).